packages/playground/package.json

```diff
@@ -56,6 +56,7 @@
     "prettier": "^3.0.3",
     "react": "^18.2.0",
     "react-dom": "^18.2.0",
+    "react-jsx-match": "^1.1.5",
```

Comment:
is this something that's already approved as a dependency?

Reply:
it was approved in gitness @abhinavrastogi-harness

Reply:
Checked it out and it was made by tan haha 
https://github.com/tnhu/react-jsx-match
@abhinavrastogi-harness

Reply:
will rewrite it tho